appveyor-windows.yml

```diff
@@ -16,11 +16,11 @@ environment:
   TEMP: D:\tmp
   TMP: D:\tmp
 
-  # MSI Installers only use Py3.6.6. It is sufficient to test with this version here.
-  PYTHON_HOME: "C:\\Python36-x64"
-  PYTHON_SCRIPTS: "C:\\Python36-x64\\Scripts"
-  PYTHON_EXE: "C:\\Python36-x64\\python.exe"
-  PYTHON_VERSION: '3.6.8'
+  # MSI Installers only use Py3.7.6. It is sufficient to test with this version here.
+  PYTHON_HOME: "C:\\Python37-x64"
+  PYTHON_SCRIPTS: "C:\\Python37-x64\\Scripts"
+  PYTHON_EXE: "C:\\Python37-x64\\python.exe"
+  PYTHON_VERSION: '3.7.5'
```

Comment:
I think this is only for linux? Do we need this? Looks unused in the file.

Reply:
It still adds visibility when looking at a appveyor test run.